packages/kbn-ecs/generated/index.ts

```diff
@@ -43,34 +53,55 @@ import { EcsTracing } from './tracing';
 import { EcsUrl } from './url';
 import { EcsUser } from './user';
 import { EcsUserAgent } from './user_agent';
+import { EcsVlan } from './vlan';
 import { EcsVulnerability } from './vulnerability';
+import { EcsX509 } from './x509';
+
+export const EcsVersion = '8.6.0' as const;
+
+/**
+ * Exporting ecs_flat schema used to generate the typings in this package, as object literal.
+ */
+export { EcsNested } from './ecs_nested';
```

Comment:
The comment above:

```ts
/**
 * Exporting ecs_flat schema used to generate the typings in this package, as object literal.
 */
```

indicates the `ecs_flat` schema will be exported, however only the `ecs_nested` schema is currently exported via:

```ts
export { EcsNested } from './ecs_nested';
```

The `ecs_flat` schema, defined by https://raw.githubusercontent.com/elastic/ecs/main/generated/ecs/ecs_flat.yml

and the `ecs_nested` schema, defined by https://raw.githubusercontent.com/elastic/ecs/main/generated/ecs/ecs_nested.yml

are structured slightly differently. The Data Quality dashboard requires the `ecs_flat` schema, where each field name is flattened (in dotted notation) to be a key in the schema, as illustrated by the screenshot below:

![ecs_flat](https://user-images.githubusercontent.com/4459398/214106665-94bc534f-a38f-4380-873b-6dbb8c7e0cd9.png)

Would you be willing to add an export for `ecs_flat`, per the example below?

```ts
export { EcsFlat } from './ecs_flat';
```

Reply:
Hey, changes made as requested :)